tests/unit/src/test/scala/tests/DefinitionLspSuite.scala

```diff
@@ -227,25 +227,20 @@ class DefinitionLspSuite extends BaseLspSuite("definition") {
           |/metals.json
           |{
           |  "a": {
-          |    "compilerPlugins": [
-          |      "org.scalamacros:::paradise:2.1.1"
-          |    ],
+          |    "scalacOptions": ["-Ymacro-annotations"],
           |    "libraryDependencies": [
-          |      "io.circe::circe-core:0.9.0",
-          |      "io.circe::circe-derivation:0.9.0-M4"
+          |       "io.github.alexarchambault::data-class:0.2.5"
           |    ]
           |  }
           |}
           |/a/src/main/scala/a/User.scala
           |package a
-          |import io.circe.derivation.JsonCodec
-          |@JsonCodec case class User(name: String)
```

Comment:
Why is the code changed here?

Reply:
So the `@JsonCodec` wasn't working the same way anymore and not actually showing an annotation the way we were testing before. So in order to still have this test show a working goto definition with an annotation I switched this to use Alex's dataclass library so we can use `@data` and see that the `@data` creates an apply that we can do a goto definition on.